java/src/org/openqa/selenium/json/JsonOutput.java

```diff
@@ -110,159 +112,169 @@ public class JsonOutput implements Closeable {
   private String indentBy = "  ";
   private boolean writeClassName = true;
 
+  /**
+   * This package-private constructor initializes new instances of the {@code JsonOutput} class.<br>
+   * This class declares methods that enable structured traversal of Java objects submitted to the
+   * {@link #write(Object)} method, serializing them as JSON strings into the {@code Appendable} object
+   * specified by [appendable].
+   *
+   * @param appendable {@link Appendable} object that receives the serialized output
+   */
   JsonOutput(Appendable appendable) {
     this.appendable = Require.nonNull("Underlying appendable", appendable);
 
     this.appender =
-        str -> {
-          try {
-            appendable.append(str);
-          } catch (IOException e) {
-            throw new JsonException("Unable to write to underlying appendable", e);
-          }
-        };
+      str -> {
+        try {
+          appendable.append(str);
+        } catch (IOException e) {
+          throw new JsonException("Unable to write to underlying appendable", e);
+        }
+      };
 
     this.stack = new ArrayDeque<>();
-    this.stack.addFirst(new Empty());
+    this.stack.addFirst(new Root());
```

Comment:
This looks suspicious.... Were things failing? Or is this part of the state-machine work?

Reply:
No, this just seems like a more descriptive name for the role this class plays.